packages/contract-tests/test/lib/utils/ens.js

```diff
@@ -48,15 +52,31 @@ describe("convertENSNames", () => {
   });
 
   it("converts ens names in address fields to addresses", async () => {
-    let result = await ens.convertENSNames(args, methodABI, Web3);
-    assert(result[0] === "0x123");
+    let result = await ens.convertENSNames({
```

Comment:
Should these tests set `networkId`?

Reply:
In this case I think "why not?". This basically will test the case where a user is connected to mainnet. Right now, if the user doesn't explicitly specify a registry address, it will use the network id to determine the address. It will fail if the address is not specified AND there is no network id. In the normal flow there will always be a network id set by Truffle. Does that make sense?

Reply:
But these tests don't seem to set either a network ID *or* a registry address.  I'm confused.  There's no `networkId` argument passed, and the `ensSettings` set above don't include a registry address either.  (They *do* include a `networkId`, but as best I can tell from the code that's ignored.)

Are these tests supposed to use network ID, or are they supposed to use registry address?  Because it seems like in neither case is the information getting passed in.

Reply:
Oh sorry, I got confused about which tests we were speaking of here. These tests stub out the `getNewENSJS` method. When you use the constructor (`require("@ensdomains/ensjs").default`) you need to feed it either a network id or address; and address takes precedence. In this case we won't instantiate and so don't need it.

Reply:
I think it would be better if it matched the real interface as closely as possible.  But I suppose it's not essential if you think it's OK.

Reply:
Oh, OK, seems this is fixed now.